interview-questions.md

```diff
@@ -7,41 +7,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer:An instance variable is when call upon a class in ruby and pass in cetain arguments that return you an object that meet the class and arguments that you called upon.  Where a variable is passed to a function to get an outcome.
 
-  Researched answer:
+  Researched answer:While in ruby an instance variable dones not have to be declared in order to be used.  However, in "regular" variable does have to be declared in order to be acted on.
```

Comment:
Okay these are close but they will be hard to translate over to working in ruby. an instance variable is a holder for data that is accessible by the whole class and therefore an "instance" of that class. these variable cannot be changed accept for setter methods, and can't be called without getter methods.